cooltools/eigdecomp.py

```diff
@@ -266,7 +266,6 @@ def trans_eig(
     A[:, is_bad_bin] = 0
 
     # Fake cis and re-balance
-    A = numutils.iterative_correction_symmetric(A)[0]
```

Comment:
this looks unnecessary , even more - it introduces chrom-sized biases in trans map with faked cis data ...

re-balancing should happen only after the cis was faked

Reply:
in the end it was problematic only for weird cases like 2-chromosomes, and or several chromosomes of very different sizes - not an issue in a "typical" scenarios .

Also , when it fails - it still preserves the checkerboard signal, however it can "place" it in EV2/3... instead